examples/ssr-example/server.js

```diff
@@ -4,15 +4,32 @@ const express = require('express')
 const server = express()
 const app = require('./index')
 
+// page template so we can hydrate the page
+// with javascript after the initial load
+const pageWithBundle = (page, src) => `
+  <div class="main">
+    ${page}
+  </div>
+  <script src="${src}"></script>
+`
+
+server.use(express.static('dist'))
+
+// serve the home page, with no bundle
+// because it needs no hydration
 server.get('/', (req, res) => {
-  res.send(app.toString('/'))
+  const page = app.toString('/')
```

Comment:
Why did you decide to break this out into two lines?

Reply:
It's not required for the home page, but is required for the other example, so I wanted to keep them consistent...

Conversely, we could have both call the `pageWithBundle` function, but that would over-complicate what is required for a page that needs no hydration